### PR TITLE
Updates to the 4.7 release notes from the tracker

### PR DESCRIPTION
**Pull request by jeana-redhat**
For items in #26801

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -347,11 +347,18 @@ You can now remove or rotate the GCP admin-level credential that the xref:../ope
 
 You can now use the Compliance Operator to perform Center for Internet Security (CIS) Kubernetes Benchmark checks. CIS profiles for {product-title} are based on the CIS Kubernetes checks.
 
+Until the CIS {product-title} Benchmark is published, you can refer to the link:https://pnt.redhat.com/pnt/p-18428165/Red_Hat_OpenS...uide_v1.0.pdf[Red Hat {product-title} Hardening Guide].
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:350]: Addressing points 1&2 of https://github.com/openshift/openshift-docs/issues/26801#issuecomment-766500209
@xiaojiey

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -347,11 +347,18 @@ You can now remove or rotate the GCP admin-level credential that the xref:../ope
...
+
 [id="ocp-4-7-security-secure-boot"]
 ==== Secure Boot support for installer-provisioned clusters
 
 You can now deploy a cluster with Secure Boot when using installer-provisioned infrastructure on bare metal nodes. Deploying a cluster with Secure Boot requires UEFI boot mode and Red Fish Virtual Media. You cannot use self-generated keys with Secure Boot.
 
+[id="ocp-4-7-cmp-acm-2-2"]
+==== Advanced Cluster Management 2.2 integration
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:358]: Addressing point 3 of https://github.com/openshift/openshift-docs/issues/26801#issuecomment-766500209
@xiaojiey

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -549,6 +556,17 @@ For more information, see xref:../machine_management/creating_machinesets/creati
 
 The Machine API now honors cluster-wide proxy settings. When a cluster-wide proxy is configured, all Machine API components will route traffic through the configured proxy.
 
+[id="ocp-4-7-mco-auto-reboot-triggers-removed"]
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:559]: Addressing https://github.com/openshift/openshift-docs/issues/26801#issuecomment-780874186
@darkmuggle 
Cc @mburke5678 FYI

File: release_notes/ocp-4-7-release-notes.adoc [hunk 4]
```diff
@@ -738,6 +756,14 @@ In {product-title} 4.7, the Insights Operator collects the following additional
 
 With this additional information, Red Hat can provide improved remediation steps in {cloud-redhat-com}.
 
+[id="ocp-4-7-auth"]
+=== Authentication and authorization
+
+[id="ocp-4-7-auth-tp-aws-sts"]
+==== Support for Amazon Web Services Security Token Service (Technology Preview)
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:763]: @joelddiaz plz give :+1: / :-1:

File: release_notes/ocp-4-7-release-notes.adoc [hunk 5]
```diff
@@ -802,7 +828,7 @@ In the table, features are marked with the following statuses:
 |Docker Registry v1 API
 |GA
 |DEP
-|
+|DEP
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:831]: @bmcelvee just FYI, and ty :)

File: release_notes/ocp-4-7-release-notes.adoc [hunk 6]
```diff
@@ -814,6 +840,11 @@ In the table, features are marked with the following statuses:
 |GA
 |DEP
 
+|Use of `v1` in `apiVersion`
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:843]: Table entry for https://github.com/openshift/openshift-docs/issues/26801#issuecomment-758051819
@soltysh

File: release_notes/ocp-4-7-release-notes.adoc [hunk 7]
```diff
@@ -831,6 +862,20 @@ When using the `oc adm catalog mirror` command to mirror catalogs, the `--filter
 
 The `--filter-by-os` flag is also now deprecated.
 
+[id="ocp-4-7-apiversion-v1"]
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:865]: Addresses https://github.com/openshift/openshift-docs/issues/26801#issuecomment-758051819
@soltysh

File: release_notes/ocp-4-7-release-notes.adoc [hunk 8]
```diff
@@ -859,6 +904,26 @@ registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.8.1
 registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.8.0
 ----
 
+[id="ocp-4-7-oc-items-removed"]
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:907]: Addresses https://github.com/openshift/openshift-docs/issues/26801#issuecomment-732152785
@ingvagabund (just 907-914)

File: release_notes/ocp-4-7-release-notes.adoc [hunk 9]
```diff
@@ -1298,6 +1363,8 @@ $ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 * Currently, the prerequisites in the web console quick start cards appear as a paragraph instead of a list. This will be resolved in a future release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905147[BZ#1905147])
 * Currently, in the *Search Page*, the *Pipelines* resources table is not immediately updated after the *Name* filter is applied or removed. However, if you refresh the page or close and expand the *Pipelines* section, the *Name* filter is applied. The same behavior is seen when you remove the *Name* filter. This will be resolved in a future release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1901207[BZ#1901207]).
 
+* Currently, on clusters with IPsec enabled, RHEL 7.9 nodes cannot communicate with RHCOS nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925925[BZ#1925925])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:1366]: Addresses https://github.com/openshift/openshift-docs/issues/26801#issuecomment-775262883#29619
@anuragthehatter

File: release_notes/ocp-4-7-release-notes.adoc (hunk 5 above)
Comment (bmcelvee) [reply release_notes/ocp-4-7-release-notes.adoc:831]: Thank you!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 6 above)
Comment (soltysh) [reply release_notes/ocp-4-7-release-notes.adoc:843]: That's not quite right, this only apply to openshift resources, iow. image streams, deployment configs, build configs, routes, etc. I'm not sure if we have somewhere a list of openshift-specific resources which we could point to.

Comment (soltysh) [reply release_notes/ocp-4-7-release-notes.adoc:843]: Probably worth pointing to https://docs.openshift.com/container-platform/4.6/rest_api/index.html and stating that every resource that has `*.openshift.io` in `apiVersion` requires that to match that value from that page and can't be just `v1`. Does that make sense?

**Review by soltysh — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-7-release-notes.adoc (hunk 6 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-7-release-notes.adoc:843]: That's a good clarification! I'll make some edits accordingly. Thanks :)

File: release_notes/ocp-4-7-release-notes.adoc (hunk 4 above)
Comment (joelddiaz) [reply release_notes/ocp-4-7-release-notes.adoc:763]: I would word it more like "Running OpenShift using AWS Security Token Service (STS) for credentials (Technology Preview)"

File: release_notes/ocp-4-7-release-notes.adoc (hunk 6 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-7-release-notes.adoc]: Updated now

File: release_notes/ocp-4-7-release-notes.adoc [hunk 10]
```diff
@@ -72,8 +72,8 @@ With `bootupd`, {op-system} users now have access to a cross-distribution, syste
 {op-system} is now using Red Hat Enterprise Linux (RHEL) 8.3 packages. {product-title} 4.6 and below will stay with RHEL 8.2 packages. This enables you to have the latest fixes, features, and enhancements, such as NetworkManager features, as well as the latest hardware support and driver updates.
 
 [id="ocp-4-7-rhcos-kdump"]
-==== {op-system} now supports kdump service (Technical Preview)
-The `kdump` service is introduced in Technical Preview in {op-system} to provide a crash-dumping mechanism for debugging kernel issues. You can use this service to save system memory content for later analysis. The `kdump` service is not managed at the cluster-level and must be enabled and configured manually on a per-node basis.
+==== {op-system} now supports kdump service (Technology Preview)
+The `kdump` service is introduced in Technology Preview in {op-system} to provide a crash-dumping mechanism for debugging kernel issues. You can use this service to save system memory content for later analysis. The `kdump` service is not managed at the cluster-level and must be enabled and configured manually on a per-node basis.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:76]: s/cluster-level/cluster level

File: release_notes/ocp-4-7-release-notes.adoc [hunk 11]
```diff
@@ -387,9 +394,9 @@ With the addition of DNS support in the OVN-Kubernetes cluster network provider
...
-==== Egress router CNI (Technical Preview)
+==== Egress router CNI (Technology Preview)
 
-The egress router CNI plug-in is introduced in Technical Preview. You can use the plug-in to deploy an egress router in redirect mode. This egress router provides parity for OVN-Kubernetes compared to OpenShift SDN, but for redirect mode only. The plug-in does not perform in HTTP proxy or DNS proxy modes, and this is a difference with the implementation for OpenShift SDN. For more information, see xref:../networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc[Deploying an egress router pod in redirect mode].
+The egress router CNI plug-in is introduced in Technology Preview. You can use the plug-in to deploy an egress router in redirect mode. This egress router provides parity for OVN-Kubernetes compared to OpenShift SDN, but for redirect mode only. The plug-in does not perform in HTTP proxy or DNS proxy modes, and this is a difference with the implementation for OpenShift SDN. For more information, see xref:../networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc[Deploying an egress router pod in redirect mode].
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:399]: .adoc#deploying-egress-router-ovn-redirection[Deploying an egress router pod in redirect mode].

File: release_notes/ocp-4-7-release-notes.adoc [hunk 12]
```diff
@@ -573,6 +580,17 @@ For more information, see xref:../machine_management/creating_machinesets/creati
...
+[id="ocp-4-7-mco-auto-reboot-triggers-removed"]
+==== Some machine configuration updates no longer cause automatic reboot
+
+The Machine Config Operator (MCO) no longer automatically reboots all corresponding nodes for the following machine configuration changes:
+
+* changes to the SSH key in the `spec.config.ignition.passwd.users.sshAuthorizedKeys` parameter of a machine config
+* changes to the global pull secret or pull secret in the `openshift-config` namespace
+* changes to the `/etc/containers/registries.conf` file, such as adding or editing an `ImageContentSourcePolicy` object
+
+For more information, see xref:../architecture/control-plane.adoc/understanding-machine-config-operator_control-plane[Understanding the Machine Config Operator].
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:592]: .adoc#understanding-machine-config-operator_control-plane[Understanding the Machine Config Operator].

File: release_notes/ocp-4-7-release-notes.adoc [hunk 13]
```diff
@@ -764,6 +782,14 @@ In {product-title} 4.7, the Insights Operator collects the following additional
...
 With this additional information, Red Hat can provide improved remediation steps in {cloud-redhat-com}.
 
+[id="ocp-4-7-auth"]
+=== Authentication and authorization
+
+[id="ocp-4-7-auth-tp-aws-sts"]
+==== Running {product-title} using AWS Security Token Service (STS) for credentials (Technology Preview)
+You can now configure the Cloud Credential Operator (CCO) to use the Amazon Web Services Security Token Service (AWS STS). When the CCO is configured to use STS, it assigns components IAM roles that provide short-term, limited-privilege security credentials.
+
+For more information, see xref:../ authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc[Support for Amazon Web Services Secure Token Service (AWS STS)].
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:792]: I don't see this file merged yet. though I could be missing something.

xref:../authentication   (extra space in the path before authentication)

The path needs a section ID added after the .adoc#

File: release_notes/ocp-4-7-release-notes.adoc (hunk 9 above)
Comment (ahardin-rh) [reply release_notes/ocp-4-7-release-notes.adoc:1433]: s/RHEL/{op-system-base-full}
s/RHCOS/{op-system-first}

These attributes are part of the common-attributes.adoc file

**Comment by ahardin-rh** [comment]
@jeana-redhat Just a few comments from me, including a few suggestions on how to fix the current build failure.

**Review by codyhoag — APPROVED** [review]
Approval exception for release notes

**Pull request merged**